sources/engine/Stride.Assets/StrideConfig.cs

```diff
@@ -236,6 +236,8 @@ internal static void RegisterSolutionPlatforms()
         internal static bool IsVSComponentAvailableAnyVersion(IDictionary<Version, string> vsVersionToComponent)
         {
             if (vsVersionToComponent == null) { throw new ArgumentNullException("vsVersionToComponent"); }
+            if (!OperatingSystem.IsWindows()) 
```

Comment:
Should this be moved before the initial null check if this is ignored anyway on other platforms?

Reply:
Yeah I can change that